sycl/CMakeLists.txt

```diff
@@ -144,7 +144,7 @@ if( NOT OpenCL_LIBRARIES )
   endif()
   ExternalProject_Add(ocl-icd
     GIT_REPOSITORY    https://github.com/KhronosGroup/OpenCL-ICD-Loader.git
-    GIT_TAG           origin/master
+    GIT_TAG           v2020.06.16
     UPDATE_DISCONNECTED ${SYCL_EP_OCL_ICD_SKIP_AUTO_UPDATE}
```

Comment:
This CMake code duplicates `dependecy.py`, so it makes sense to call `dependency.py` from here to avoid duplication.

Reply:
I would prefer to eliminate dependency.py instead to match build process recommended in [Get started guide](sycl/doc/GetStartedGuide.md#build-dpc-toolchain).

Reply:
> I would prefer to eliminate dependency.py instead to match build process recommended in [Get started guide](sycl/doc/GetStartedGuide.md#build-dpc-toolchain).

In both cases there are no changes in "build process recommended in [Get started guide](sycl/doc/GetStartedGuide.md#build-dpc-toolchain)". Having `dependency.py` simplifies dependencies configuration process, but I don't have a strong preference here because it's just one library.